belgif-rest-problem/src/main/java/io/github/belgif/rest/problem/api/InputValidationIssue.java

```diff
@@ -124,41 +134,49 @@ public Object getValue() {
     }
 
     public void setValue(Object value) {
-        verifyNoInputs();
+        verifyNoInputs(value);
         this.value = value;
     }
 
     public List<Input<?>> getInputs() {
         return Collections.unmodifiableList(inputs);
     }
 
-    public void setInputs(List<Input<?>> inputs) {
```

Comment:
Should we remove them now, or first mark them `@Deprecated(forRemoval = true)` and only remove them when going to 1.0?

The inputs[] is not belgif-standard, it's a CBSS extension.
So I guess the risk is low that anyone outside of CBSS would be impacted by this removal, and removing them immediately should not be too problematic?

Reply:
> Should we remove them now, or first mark them `@Deprecated(forRemoval = true)` and only remove them when going to 1.0?

It does not seem to be used by any project (I did not check the PR though) so I don't think it's a problem to remove it.